tests/block_test.go

```diff
@@ -75,8 +69,11 @@ func (suite *ExecutionSpecBlockTestSuite) TestExecutionSpecBlock() {
 	bt.skipLoad(`^prague\/eip7685_general_purpose_el_requests`)
 	bt.skipLoad(`^prague\/eip7002_el_triggerable_withdrawals`)
 	bt.skipLoad(`^prague\/eip6110_deposits`)
+	// different amount of gas is consumed because 0x0b contract is added to access list by ActivePrecompiles although Cancun doesn't have it as a precompiled contract
+	bt.skipLoad(`^frontier\/precompiles\/precompile_absence\/precompile_absence.json`)
```

Comment:
Please run Prague cases specifying more detail like [this](https://github.com/kaiachain/kaia/blob/04176c6b2b4477fd49d1df55c7668e75a2f1eb70/tests/state_test.go#L101-L103).

Reply:
I've fixed in https://github.com/kaiachain/kaia/pull/262/commits/da88a24266472ecab4c1820e84656238b82193be. Thank you.